### PR TITLE
[CloudPC]Add virtualNetworkLocation to onPremisesConnection

### PR DESCRIPTION
**Pull request by hyc3z**
Roadmap Task Link:
https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15680/

### CODE REVIEW

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@hyc3z 
1.Please also update the `changelog/Microsoft.CloudManagedDesktop.json` file.
2.In the description you only need to add the roadmap workitem number. In this work item, you can describe the review PR and graph schema PR link

**Comment by hyc3z** [comment]
Updated @RuiHou105 please have a look, thanks

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 1]
```diff
@@ -2573,6 +2573,114 @@
             "CreatedDateTime": "2022-08-24T02:53:28.597151Z",
             "WorkloadArea": "Devices and apps",
             "SubArea": "Cloud PC"
+        },
+        {
+            "ChangeList": [
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:2578]: @hyc3z  
As your change only includes `virtualNetworkLocation` Property, so please only include this one in changelog: 
image

**Comment by hyc3z** [comment]
Updated

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -2601,6 +2601,24 @@
...
             "SubArea": "Cloud PC"
         },
+        {
+            "ChangeList": [
+              {
+                "Id": "0761fa9b-d6c7-41cb-8f1c-749379d1055d",
+                "ApiChange": "Property",
+                "ChangedApiName": "virtualNetworkLocation",
+                "ChangeType": "Addition",
+                "Description": "Added the `virtualNetworkLocation` property to the [cloudPcOnPremisesConnection](https://docs.microsoft.com/en-us/graph/api/resources/intune-cloudPcOnPremisesConnection?view=graph-rest-beta) resource.",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:2611]: The link for cloudPcOnPremisesConnection is wrong. Could you please help fix it?

**Review by RuiHou105 — APPROVED** [review]

**Comment by Danielabom** [comment]
Contacted author to get info.

File: api-reference/beta/resources/cloudpconpremisesconnection.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ Represents a defined collection of Azure resource information that can be used t
 |inUse|Boolean|When `true`, the Azure network connection is in use. When `false`, the connection is not in use. You cannot delete a connection that’s in use. Returned only on `$select`. For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
 |managedBy|[cloudPcManagementService](#cloudpcmanagementservice-values)|Specifies which services manage the Azure network connection. Possible values are: `windows365`, `devBox,` `unknownFutureValue`. Read-only.|
 |organizationalUnit|String|The organizational unit (OU) in which the computer account is created. If left null, the OU that’s configured as the default (a well-known computer object container) in your Active Directory domain (OU) is used. Optional.|
+|virtualNetworkLocation|String|Indicates resource location of the target virtual network(Readonly, computed value).|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:47]:
```suggestion
|virtualNetworkLocation|String|Indicates resource location of the virtual target network (Read-only, computed value).|
```

File: api-reference/beta/resources/cloudpconpremisesconnection.md [hunk 4]
```diff
@@ -44,6 +44,7 @@ Represents a defined collection of Azure resource information that can be used t
 |inUse|Boolean|When `true`, the Azure network connection is in use. When `false`, the connection is not in use. You cannot delete a connection that’s in use. Returned only on `$select`. For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
 |managedBy|[cloudPcManagementService](#cloudpcmanagementservice-values)|Specifies which services manage the Azure network connection. Possible values are: `windows365`, `devBox,` `unknownFutureValue`. Read-only.|
 |organizationalUnit|String|The organizational unit (OU) in which the computer account is created. If left null, the OU that’s configured as the default (a well-known computer object container) in your Active Directory domain (OU) is used. Optional.|
+|virtualNetworkLocation|String|Indicates resource location of the virtual target network (Read-only, computed value).|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:47]:
```suggestion
|virtualNetworkLocation|String|Indicates resource location of the virtual target network. Read-only, computed value.|
```

**Review by Danielabom — APPROVED** [review]
@hyc3z , thank you for working on this PR. Everything looks good. PR approved.

**Pull request merged**